docs/api-operation.md

```diff
@@ -169,7 +169,7 @@ This operation will always occur after resizing and extraction, if any.
 // to the top, left and right edges and 20 to the bottom edge
 sharp(input)
   .resize(140)
-  .background({r: 0, g: 0, b: 0, a: 0})
+  .background({r: 0, g: 0, b: 0, alpha: 0})
```

Comment:
This change was included in commit 4730554 so I think this file can be removed from the PR.